core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -63,6 +63,19 @@ public static boolean ancestorOf(Table table, long snapshotId, long ancestorSnap
     return ancestorIds(table.currentSnapshot(), table::snapshot);
   }
 
+  /**
+   * Traverses the history of the table's state and finds the oldest Snapshot.
```

Comment:
I think that "table's state" isn't clear enough. How about "history of the table's current snapshot" like the one below?

Reply:
fixed.